synapse/lib/lmdblayer.py

```diff
@@ -213,6 +201,32 @@ class LmdbLayer(s_layer.Layer):
         if univ:
             self.layrslab.delete(penc + oldi, pvvalu, db=self.byuniv)
 
+    async def _storSplices(self, splices):
+        self.splicelog.save(splices)
+
+    async def _storProvStack(self, prov):
+        data = s_msgpack.en(prov)
```

Comment:
since stack frames contain dicts, we may want a special hash calc that makes them key ordered list of tuples?  i worry about stable ordering in python dicts making the iden it generates sensitive to argument order in claim()

Reply:
ok. It'd be easier just to convert it to tuples higher up.

Reply:
hmm maybe not